helm-flx.el

```diff
@@ -148,7 +148,7 @@ Return candidates prefixed with basename of `helm-input' first."
     (if (string= helm-pattern "")
         candidates
       (if (string-match-p " " helm-pattern)
-          (helm-fuzzy-matching-default-sort-fn candidates source use-real)
+          (helm-fuzzy-matching-default-sort-fn candidates source)
         (helm-flx-sort candidates
                        helm-pattern
                        (if use-real
```

Comment:
Not sure which instance you mean.  If you mean line 154 `(if use-real`, I didn't touch that because it's an argument to `helm-flx-sort`; if we remove that, it might break `helm-flx-sort`.  If you mean the argument in line 146, I didn't touch that, because if we remove it, it will break anything that calls this function--i.e. the same kind of breakage we're fixing.